docs/user/guides/automation-rules.rst

```diff
@@ -14,10 +14,12 @@ Automating your versioning on Read the Docs means you only have to handle your v
    :doc:`/automation-rules`
      Reference for all different rules and actions possible with automation.
 
-Adding a new rule
------------------
+Adding a new Automation Rule
```

Comment:
We should agree on capitalization of this term, since we use it both ways.

Reply:
Yes, I find that the prevailing "case" is lower case. I sometimes upper case per reflex because the UI in the Dashboard does that. But it's unintended.

Reply:
Confirmed that a lot of other places are lowercase and made this lowercase, too.